crates/suiop-cli/Cargo.lock

```diff
@@ -1051,7 +1051,7 @@ checksum = "3e2e65a1a2e43cfcb47a895c4c8b10d1f4a61097f9f254f183aee60cad9c651d"
 
 [[package]]
```

Comment:
why is there a lockfile here? either way these changes need to be reverted as you can't change a lockfile like this

Reply:
hmm, i did not touch file. Maybe it got modified by `cargo build --release`

Reply:
Just reverted the changes